### DISCUSSION TITLE
Acces the actual entry in a matrix field _partial

### DISCUSSION THREAD

**Discussion by FrDH** [craftcms/cms#15842] [Q&A] [answered]
How can I acces the actual entry (the "page") inside a _partial template?
For example, the template "_partials/entry/faq" is automatically included when the "faq" (matrixfield) entry type is used in a ck-editor field. In the template, the `entry` variable points to the matrixblock, `entry.owner` points to the ck-editor and `entry.owner.owner` points to the "page".
Since the nesting level can be infinite, is there a good way to access the page? I currently have this in my "_partials/entry/faq" template :
```
    {% set owner =
        entry.owner.owner.owner.owner.owner.owner ???
        entry.owner.owner.owner.owner.owner ???
        entry.owner.owner.owner.owner ???
        entry.owner.owner.owner ???
        entry.owner.owner ???
        entry.owner
    %}
```

Not my proudest piece of Twig...

**Comment by mmikkel** [comment, marked as answer]
You're looking for `getRootOwner()`, which was [added in Craft 5.4](https://github.com/craftcms/cms/commit/b1e4385de04b01d52ebdf28df58af1920296099a):

```twig
{% set owner = entry.getRootOwner() %}
```

**Comment by FrDH** [comment]
Thank you @mmikkel (I was on 5.3 but after updating all worked well)
